### PR TITLE
Basic handling of message-type req

### PR DESCRIPTION
**Pull request by forsberg**
Basic handling of message-type req. Not at all fancy, not using queue or anything, so I might have missed some important requirement.

### CODE REVIEW

File: mysensors/mysensors.py [hunk 1]
```diff
@@ -368,6 +376,12 @@ def set_child_value(self, child_id, value_type, value):
                             sub_type=value_type, payload=value)
         return None
 
+    def get_child_value(self, child_id, value_type):
```
Comment (MartinHjelmare) [inline mysensors/mysensors.py]: I don't think you need this method. You have access to the values in children in the sensors attribute in the Gateway class.

File: mysensors/mysensors.py [hunk 2]
```diff
@@ -62,6 +62,12 @@ def _handle_set(self, msg):
                 msg.child_id, msg.sub_type, msg.payload)
             self.alert(msg.node_id)
 
+    def _handle_req(self, msg):
+        if self.is_sensor(msg.node_id, msg.child_id):
+            value = self.sensors[msg.node_id].get_child_value(msg.child_id, msg.sub_type)
+            if value:
+                return msg.copy(type=MessageType.set, payload=str(value))
```
Comment (MartinHjelmare) [inline mysensors/mysensors.py]: The value should already be stored as a string, so you don't need to convert it. I've removed the global MessageType on the dev branch. There you can use self.const.MessageType.set, in the Gateway class.

**Comment by MartinHjelmare** [comment]
Hi!

I think we should keep development on the dev branch, so you should rebase off of dev branch. You might have to force push after that. If you can have a look at my comments and make those changes, I think we're good to go.

**Comment by forsberg** [comment]
There you are - rebased off dev branch, and modified according to your comments.

**Comment by MartinHjelmare** [comment]
Great! Can you add docstrings to the four new methods as well? Sorry I missed that before. Then you have to make a new pull request with your branch and point it to the dev branch here instead of master. I want to add a couple of features/fixes before we bump version.

Thanks for your contribution! It's great to see more and more people are using home assistant and mysensors, and want to make them better.

**Pull request closed without merging**